module3/assignment3.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Assignment 3</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    <link rel="stylesheet" href="_css/larson_a3.css" />
+</head>
+<body>
+<!-- Owen Larson
+     assignment3.html
+     INFO2439
+     Thoendel
+     9/29/19
+-->
+<div data-role="page">
+<div data-role="header">
+    <a data-ajax="false" href="https://student.mccinfo.net/~owlarson/" class="ui-btn-left" target="_blank" data-icon="home">Home</a>
```

Comment:
You wouldn't need an absolute reference here - a relative - or better yet - dynamic back/home button would be preferred.